Makefile

```diff
@@ -232,6 +232,7 @@ localunit: test/goecho/goecho varlink_generate
 		--succinct
 	$(MAKE) -C contrib/cirrus/packer test
 	./contrib/cirrus/lib.sh.t
+	./contrib/cirrus/cirrus_yaml_test.py
```

Comment:
Is running this here okay?  I was going to add it to the gating tests originally, but second-guessed myself.  I'm not opposed to moving it and even `lib.sh.t` since they're both very specific to CI and not building itself.